org.adempiere.base.process/src/org/compiere/process/M_PriceList_Create.java

```diff
@@ -117,10 +117,7 @@ private String checkPrerequisites() {
 				"UPDATE M_Product_PO SET PriceLastPO = 0 WHERE PriceLastPO IS NULL AND AD_Client_ID=?",
 				clientParam, get_TrxName());
 		DB.executeUpdateEx(
-				"UPDATE M_Product_PO SET PricePO = PriceLastPO WHERE (PricePO IS NULL OR PricePO = 0) AND PriceLastPO <> 0 AND AD_Client_ID=?",
-				clientParam, get_TrxName());
-		DB.executeUpdateEx(
-				"UPDATE	M_Product_PO SET PricePO = 0 WHERE PricePO IS NULL AND AD_Client_ID=?",

```

Comment:
This UPDATE is important, why it was deleted?

Reply:
its function is included in the update before. PriceLastPO is always defined two updates earlier so it could be assigned to PricePO even if it's not non zero. 
The enhencement of this is so small that I didn't find it worth a seperate ticket. I could remove it, if requested.

Reply:
yes - I think is better to remove this - still not convinced the change is harmless because the WHERE there seems wrong
-> WHERE PricePO IS NULL OR (PricePO = 0 AND PriceLastPO <> 0) AND AD_Client_ID=?

is hard to follow OR and AND without proper parenthesis, so, first, that WHERE can be updating records from a different clientID.

I would suggest to drop these two changes to the UPDATEs that are not adding value - but made the peer review more complex (hard to follow what is intended here).

Reply:
Ok, I'll change that. And you are right: there is a missing bracket which will lead to all PricePO which are NULL will be updated to PriceLastPO even if not in the current client. It wouldn't do harm, but it's wrong.